pkgs/misc/emulators/dolphin-emu/master.nix

```diff
@@ -66,18 +66,18 @@ in stdenv.mkDerivation rec {
   postInstall = ''
     cp -r ${desktopItem}/share/applications $out/share
     ln -sf $out/bin/dolphin-emu $out/bin/dolphin-emu-master
-  '' + stdenv.lib.optionalString stdenv.isLinux ''
+  '' + lib.optionalString stdenv.isLinux ''
     wrapProgram $out/bin/dolphin-emu-nogui \
```

Comment:
Since this is being wrapped already `wrapQtAppsHook` is going to skip it since it only wraps elf executables.

What we can do is
```
qtWrapperArgs = lib.optionals stdenv.isLinux [
  "--prefix LD_LIBRARY_PATH : ${vulkan-loader}/lib"
];
```